src/flb_log.c

```diff
@@ -57,10 +58,78 @@ static inline int consume_byte(flb_pipefd_t fd)
     return 0;
 }
 
+static inline bool log_file_sz_max(off_t size, struct flb_log *log)
+{
+    double sz_mb;
+
+    sz_mb = size / (1024.00 * 1024.00);
+    if (sz_mb >= log->max_sz_mb) {
+        return true;
```

Comment:
use FLB_TRUE/FLB_FALSE instead (and change return type to int)

Reply:
updated.